index.html

```diff
@@ -39,36 +39,35 @@
 <header>
   <div id="header-inner">
     <a href="#" id="logo"></a>
-     <a class="navbar-brand1" rel="home" href="#" title="logo">
-            <img src="src/assets/logo.png" width="90" height="80" >
-    <div class="container1 blue circleBehind">
-        <a href="#" id="menu-icon"></a>
-        <a href="#" class="current">Home</a>
-        <a href="sort.html">Sorting</a>
-        <a href="search.html">Searching</a>
-        <a href="#">Path Finding</a>
-        <a href="#">Graph</a>
-		<div class="dropdown">

```

Comment:
@2024-SANDHYA this dropdown menu was added in the PR that was last merged, you should not reject it.